common/men_at_arms_types/00_cultural_maa_types.txt

```diff
@@ -1 +1 @@
-﻿# Warcraft
+﻿# Warcraft
```

Comment:
Just noticed, you should adapt `00_culture_eras.txt` to our timeline.

Reply:
not really sure how this works, different cultures will unlock eras at different times, I updated it but I'm not sure this is what you're looking for.

Reply:
> not really sure how this works, different cultures will unlock eras at different times, I updated it but I'm not sure this is what you're looking for.

Oops, I haven't noticed you were already adapted.

Though `culture_era_late_medieval` is too close to `culture_era_high_medieval`. In vanilla, there's 150 years gap.